src/CMakeLists.txt

```diff
@@ -26,9 +26,11 @@ target_include_directories(VulkanMemoryAllocator PUBLIC
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
 )
 
-# Only link to Vulkan if static linking is used
+# Only link to Vulkan library if static linking is used, but always add Vulkan headers directory
 if(${VMA_STATIC_VULKAN_FUNCTIONS})
     target_link_libraries(VulkanMemoryAllocator PUBLIC Vulkan::Vulkan)
+else()
+    target_include_directories(VulkanMemoryAllocator PUBLIC "${Vulkan_INCLUDE_DIRS}")
```

Comment:
Shouldn't this be `target_link_libraries(VulkanMemoryAllocator PUBLIC Vulkan::Headers)`?